### DISCUSSION TITLE
No Windows build for 1.5.3?

### DISCUSSION THREAD

**Discussion by robsonhermes** [diasurgical/DevilutionX#7383] [Q&A] [answered]
Hello =)

Sorry if I'm missing anything, but I cannot find Windows build for 1.5.3. Should I build from the source?

Best
Robson

**Comment by StephenCWills** [comment, marked as answer]
Preparation for that build is lagging behind the others. Give it a couple more days.

**Comment by AJenbo** [comment]
The 64bit Windows build is up now

**Reply by robsonhermes** [reply]
@AJenbo thanks a lot! And thanks for keeping D1 alive thru devilutionX!
